android-kotlin/app/src/main/java/com/mongodb/app/Task.kt

```diff
@@ -0,0 +1,12 @@
+package com.mongodb.app
+
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+import org.bson.types.ObjectId
+
+open class Task(
+        @PrimaryKey var _id: ObjectId = ObjectId(),
+        var _partition: String = "",
```

Comment:
I assume this is the partition key. If so, this isn't needed... It is inserted automatically by the protocol